dbt/adapters/athena/connections.py

```diff
@@ -169,6 +171,10 @@ def open(cls, connection: Connection) -> Connection:
                         "InternalServerException",
                     ),
                 ),
+                config=config.Config(
+                    user_agent_extra="dbt-athena-community/"
+                    + pkg_resources.get_distribution("dbt-athena-community").version
+                ),
```

Comment:
they way how we create the config here and inside impl.py is the same, could you centralize this in a little util and call that instead in both places?